www/src/components/plugin-searchbar-body.js

```diff
@@ -18,7 +18,7 @@ import unescape from "lodash/unescape"
 import presets from "../utils/presets"
 import typography, { rhythm, scale } from "../utils/typography"
 import { scrollbarStyles } from "../utils/styles"
-import { css as glam } from "glamor"
+import { injectGlobal } from "emotion"
```

Comment:
Hmm, wondering if we should always import from [react-emotion](https://emotion.sh/docs/react-emotion)?

/cc @DSchau

Reply:
You can savely use react-emotion everywhere as emotion is included with that

Reply:
👍 ☝️ @janczizikow Could you change those imports from `emotion` to `react-emotion` please? 🙏 

(Thanks @LeKoArts!)

Reply:
What @LeKoArts said ☝️ react-emotion just exports everything from emotion!

Reply:
@fk Sure, no problem 🙂